src/click-outside.directive.ts

```diff
@@ -115,14 +125,26 @@ export class ClickOutsideDirective implements OnInit, OnChanges, OnDestroy {
     }
 
     if (!this._el.nativeElement.contains(ev.target) && !this._shouldExclude(ev.target)) {
-      this._ngZone.run(() => this.clickOutside.emit(ev));
+      this._emit(ev);
 
       if (this.attachOutsideOnClick) {
         this._events.forEach(e => document.body.removeEventListener(e, this._onClickBody));
       }
     }
   }
 
+  private _onWindowBlur(ev: Event) {
```

Comment:
Can you put a comment above this just so that it's more obvious what it's for (basically a tl;dr of the original GitHub issue, and maybe a link to the issue).